src/pages/signin/SignInPage.js

```diff
@@ -28,6 +29,9 @@ const propTypes = {
 
         /** Whether the account is validated */
         validated: PropTypes.bool,
+
+        /** The primaryLogin associated with the account */
+        primaryLogin: PropTypes.string,
```

Comment:
I think we should add a default prop for this

Reply:
Hmm, why? The other values don't have a default prop associated with them.

Reply:
I think we either make the prop required else we define the default prop

Reply:
Fair, but neither `errors` nor `validated` have default props here (nor in most other cases from what I can see). I'm happy to add default props for all of them if you'd like, but also to content to leave the default props as is.

Reply:
Yeah, I think there was a lint rule that required default props or the prop to be required. Looks like it is not working lately.